wp/wp-content/themes/lahma_maker/template-parts/content-arcsi.php

```diff
@@ -110,10 +116,10 @@ function sortAssociativeArrayByKey($array, $key, $direction){
         <h4 class="episode-name"><?php echo $showname; ?></h4>   
         <p><?php echo $showdescription; ?></p> 
         <div id="arcsi-audio-<?php echo $showid; ?>" class="arcsicontrols">
-            <a class="arcsibutton arcsidown" href="<?php echo $server; ?>/arcsi/item/<?php echo $showid; ?>/download" target="_blank">
+            <a class="arcsibutton arcsidown avoidAjax" href="#" data-href="<?php echo $server; ?>/arcsi/item/<?php echo $showid; ?>/download" title="<?php echo $fullTitle; ?>">
```

Comment:
What's the reason of the change? I tried both and didn't notice any difference (download opens a new tab, which closes immediately).

Reply:
yes, this is how it works only, if it is a href link, the browser prioritize it and doesn't send the gtag, because it already opens a new window.
so the process how it works is like this arcsidown triggers the `arcsiDownload()` function and sends in the title for gtag and the data-href link for the download. the function sends the gtag, and then opens the download link

Reply:
I see. nice!